builders/get-started/endpoints.md

```diff
@@ -59,3 +60,15 @@ To create a custom OnFinality endpoint, go to [OnFinality](https://onfinality.io
 3. Your custom API endpoint will be generated automatically
 
 ![OnFinality](/images/builders/get-started/endpoints/endpoints-2.png)
+
+### Dwellir {: #dwellir }
+
+[Dwellir](https://dwellir.com/development-nodes/){target=_blank} provides free endpoints for developers and dedicated nodes on our owned and operated bare-metal cluster. We're building custom data solutions and tooling so get in touch.
+
+To get started go to [Dwellir](https://dwellir.com/development-nodes/){target=_blank}:
+
+1. Click on **Get your Node**
```

Comment:
Not sure if you prefer users to go to your main website then click on **Get your Node** or if you want them to be automatically sent to the development-node page... but as the instructions stand, users don't have to click on **Get your Node** because they are already on the correct page so this can be updated one way or another

Reply:
I have just updated the instructions